docs/manuals/mapping.md

```diff
@@ -19,7 +23,19 @@ mapping experience.
       You may have to enable installing from unknown sources in your device
       settings too.
 
-### Step 2: Access the Mapper Frontend
+### Step 2: Add metadata on odk application(Optional)
```

Comment:
I think this section should be removed, as its not something the user should have to configure manually. All the fields you mention are already included by default.

Sure we don't actually inject an email into the field yet (as its not available, we may never do this), but its enabled anyway

Reply:
But I am only inform the users that if their project needs email address, phone number or any other metadata that we don't have, they don't need to fill up in every form rather they can use this method. Otherwise its not compulsory and completely optional behavior for them. 
However, some projects need their user's phone number to contact them if needed. In such cases, they can add it in instruction and this document can help was my point of view.

Reply:
But we have no ability to configure this with the current code: the fields are always enabled by default. Perhaps this would make sense if we added the config option

Reply:
Maybe I'm misunderstanding this, as I have never tested the approach you mention